app/gui/src/components/TabView.vue

```diff
@@ -147,4 +163,30 @@ const onSignOut = () => {
   min-height: 0;
   display: flex;
 }
+
+.closeButton {
+  --button-padding: 0px;
+  --icon-width: 12px;
+  --icon-height: 12px;
+
+  &:hover {
+    background-color: rgba(239, 68, 68, 0.8);
+  }
+}
+
+.App.onMacOs .closeButton {
```

Comment:
Using styles like these you're entering the danger zone of:
1. Copy-pasting styles (they are _not_ reusable) 
2. Style overrides and specificity calculations.

Reply:
As a rule, I assume if anyone would need to use same styles, they will generalize them (this is not hard to do in this case). But I may also make CloseButton component as in rect.

As for specificity calculations... these are scoped styles, so I guess it's not something unmanageable?

Reply:
Scoped styles have nothing to do with the specificity, it's only encapsulation.

FWIW On dashboard we use tailwind + tailwind-variants to make styles with constant specificity(0,1,0) and a great customization

Reply:
What I meant is that having styles scoped to a single component, it's easier to handle all cases handled to specificity - it's not like your stile will hover something set in a completely different place.

Reply:
Agree it's easiear, but doesn't solve the specificity issue.

Reply:
Well, I assume, unless we would start making many very specific global css selectors, it shouldn't be problem at all? The scoped always would have priority and inside a single component specificity should be manageable. In Vue we never had problems with it.